cli/run.sh

```
@@ -13,5 +13,10 @@ then
   #echo $(which $FP_TSC) $@
   exec $(which $FP_TSC) $@
 fi
-
-exec pnpm exec tsx $(dirname $0)/main.ts $@
+dirName=$(dirname $0)
+if [ -f $dirName/main.js ]
+then
+   exec node $dirName/main.js $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)